File: taskrouter/v1/workflow/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

workflow = @client.taskrouter.workspaces('WSaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                             .workflows
                             .create(
                                friendly_name: 'friendly_name',
                                configuration: 'configuration'
                              )

puts workflow.sid

File: trunking/v1/origination_url/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Converters;
using Twilio.Rest.Trunking.V1.Trunk;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var originationUrl = OriginationUrlResource.Create(
            pathTrunkSid: "TRaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            weight: 1,
            priority: 1,
            enabled: true,
            friendlyName: "FriendlyName",
            sipUrl: new Uri("https://example.com")
        );

        Console.WriteLine(originationUrl.AccountSid);
    }
}